app-drawer/app-drawer.html

```diff
@@ -108,24 +126,41 @@
     }
 
     #scrim {
-      position: fixed;
-      top: -120px;
+      position: absolute;
+      top: 0;
+      bottom: 0;
       left: 0;
       right: 0;
-      bottom: -120px;
       background: var(--app-drawer-scrim-background, rgba(0, 0, 0, 0.5));
       opacity: 0;
-      transition: opacity 0.2s ease;
+      visibility: hidden;
```

Comment:
Do we need to set `visibility: hidden` on #scrim?  Seems like we don't need to do that here and also L159.